File: Tema_1/CalculoSuma1File.f90
!Programa en Fortran 90 almacenado 
!en un unico fichero
program CalculoSuma
  implicit none
  real:: x, y, result, suma
  write(*,*)'Introduce el primer sumando'
  read(*,*)x
  write(*,*) 'Introduce el segundo sumando'
  read(*,*)y
  result = suma(x,y)
  write(*,*) 'El resultado es: ', suma
end program CalculoSuma
!Fin del programa principal
function suma(x,y)
  implicit none
  real:: suma, x, y
  suma = x+y
  return
end function suma

File: Tema_1/CalculoSuma1File.cpp
/*Programa sencillo en C++
escrito en un solo fichero*/
//Cabeceras
#include <iostream>
using namespace std;

//Prototipos de funciones
double suma(double, double);

//Variables globales
double x, y;

//Parte principal del programa
int main() //Funcion especial con el identificador de main
{
  double result;
  cout<<"Introduce el primer sumando";
  cin>>x;
  cout<<"Introduce el segundo sumando";
  cin>>y;
  result = suma(x,y);
  cout << "El resultado es: \n";
  cout <<result;
  return 0;
}

//Definición de la funcion suma
double suma(double x1, double y1)
{ //variable local
  double result;
  result = x1 + y1;
  return(result);
}

